src/common/Types.ts

```diff
@@ -1148,9 +1149,15 @@ export interface ClipboardEvent extends SyntheticEvent {
 
 export type FocusEvent = SyntheticEvent;
 
+export enum MouseButton {
+    Primary,
```

Comment:
Please remove this enumeration since its use in MouseEvent will result in a breaking change.